src/main/java/trackr/model/commons/Name.java

```diff
@@ -10,12 +10,12 @@
 public abstract class Name {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "%s names should only contain alphanumeric characters and spaces, and it should not be blank";
+            "%s names should only contain alphabetical characters and spaces, and it should not be blank";
 
     /*
      * The first character of the name must not be a whitespace, otherwise " " (a blank string) becomes a valid input.
      */
-    private static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    private static final String VALIDATION_REGEX = "^[a-zA-Z ]+$";
```

Comment:
Please revert back to the original regex as this will affect all `XYZName` and their test cases. It also makes sense to have alphanumeric in `TaskName` ie “Buy 10 eggs"

Reply:
Noted. I have changed it back